content/03.methods.md

```diff
@@ -11,10 +11,50 @@
 2. Describe metadata statistics on PubMed central (number of articles, how many articles were processed
 
 ### Comparing Corpora
-1. Spacy to process text via - Lemmatization, removal of stop words
-2. Describe counting frequencies of each lemma
-3. Describe using chi-square test
-4. Describe how to calculate the likelihood and log odds ratio
+The first step in linguistic analysis is to preprocess the text for ease of computation.
+This step consists of removing non-informative words (i.e., stop words) as well as converting words into their root form (i.e., lemmatization).
+Throughout our analysis we encountered symbols that aren't considered words (e.g., $\pm$), so we will refer to words and symbols as tokens to avoid confusion.

```

Comment:
```suggestion
Throughout our analysis we encountered non-word symbols (e.g., $\pm$), so we refer to words and symbols as tokens to avoid confusion.
```